src/main/java/uk/gov/hmcts/reform/slc/logging/AppInsights.java

```diff
@@ -94,8 +96,29 @@ public void markMessageNotHandled(String messageId, java.time.Duration duration)
         );
     }
 
-    public void trackMessageMappedToLetter(String messageId, String serviceName, String template, long bodyLength) {
+    public void trackLetterNotHandled(Letter letter) {
         Map<String, String> properties = ImmutableMap.of(
+            "letterId", letter.id.toString(),
+            "service", letter.service,
+            "type", letter.type
```

Comment:
do we need to add messageid?

Reply:
can add but since we going to retire the service bus integration plus more importantly we need `letterId` in reports i think it's ok to start skipping that property